### ISSUE TITLE
PAT Cloud not able to download detailed results

### ISSUE THREAD

**Issue by axelstudios** [NREL/OpenStudio#1723]
Cloud simulations end normally if you don't have any "detailed download" options selected.  However, if you do, the simulation ends but appears to still be running, and the detailed results will never be downloaded.  The Pat log shows many lines of the following:

```
[openstudio.analysisdriver.CloudAnalysisDriver] <-1> Waiting on detailed results for 4 DataPoints.
[openstudio.analysisdriver.CloudAnalysisDriver] <-2> Received reply to request for data points ready for download. There are 0.
[openstudio.analysisdriver.CloudAnalysisDriver] <-1> Waiting on detailed results for 4 DataPoints.
[openstudio.analysisdriver.CloudAnalysisDriver] <-2> Received reply to request for data points ready for download. There are 0.
[openstudio.analysisdriver.CloudAnalysisDriver] <-1> Waiting on detailed results for 4 DataPoints.
[openstudio.analysisdriver.CloudAnalysisDriver] <1> No points became ready for download within 60 tries.
```

**Comment by asparke2** [comment]
PAT 1.X issue, no longer relevant.
